components/ComicPanels.vue

```diff
@@ -6,27 +6,45 @@
             type: Object,
             default: () => {},
         },
-        selectedId: {
-            type: String,
-            default: () => null,
-        },
         lockAspectRatio: Boolean,
     });
-
-    const stripHeight = ref(0);
     const activePanelIndex = ref(0);
     const comicStore = useComicStore();
+    const panelBBArray = reactive([]);
+    props.comic
+        .getPage(0)
+        .getStrip(0)
+        .panels.forEach((panel, index) => {
+            panelBBArray.push({ height: props.comic.getPage(0).getStrip(0).height, width: panel.width });
+        });
+    let DOMElementArray;
+
+    onMounted(() => {
+        DOMElementArray = Array.from(document.getElementsByClassName('swiper-no-swiping'));
+        updatePanelBB();
+    });
+
+    window.onresize = updatePanelBB;
+
+    function updatePanelBB() {
+        if (panelBBArray.length !== DOMElementArray.length) return;
+
+        DOMElementArray.forEach((DOMElement, index) => {
+            panelBBArray[index].height = DOMElement.getBoundingClientRect().height;
+            panelBBArray[index].width = DOMElement.getBoundingClientRect().width;
+            props.comic.getPage(0).getStrip(0).panels[index].width = toRaw(panelBBArray[index]).width;
+            props.comic.getPage(0).getStrip(0).height = toRaw(panelBBArray[index]).height;
+        });
+    }
```

Comment:
I am not sure if I fully understand this function. Shouldn't the Panel  width be set in the WrapperCanvas.